build.xml

```diff
@@ -540,6 +540,7 @@
             <exclusion groupId="com.google.guava" artifactId="guava"/>
           </dependency>
           <dependency groupId="org.apache.cassandra" artifactId="dtest-api" version="0.0.11" scope="test"/>
+          <dependency groupId="org.apache.cassandra" artifactId="harry-core" version="1.0.0-SNAPSHOT" scope="test"/>
```

Comment:
Looks like this artifact hasn't been uploaded yet, my build fails with this error and I can't find it on Maven Central.

```
BUILD FAILED
~/Repos/apache/cassandra/.build/build-resolver.xml:174: The following error occurred while executing this line:
~/Repos/apache/cassandra/.build/build-resolver.xml:72: Task [resolver:resolve] failed after [3] attempts; giving up.
Error messages:
Could not resolve artifacts: Could not find artifact org.apache.cassandra:harry-core:jar:1.0.0-SNAPSHOT
```

Reply:
@aratno this is weird, since I'm pretty positive it actually got published: https://repository.apache.org/content/repositories/snapshots/org/apache/cassandra/harry-core/

It's not on maven central, but that's probably an artifact of how mirroring works for apache repos. 

I've pushed a fix that should make sure snapshots are downloaded unless we're on release (at least for review purposes).

Reply:
Works now! Thanks for fixing.